docs/backend/upgrading/version-specific-migration/upgrade-to-60.md

```diff
@@ -650,3 +650,8 @@ Plone 6.0 makes changes to two viewlet managers:
 One final change is that Plone 6.0 moves the `plone.footer` viewlet from `plone.app.layout/viewlets` to `plone.app.portlets`.
 The viewlet remains in manager `plone.portalfooter`.
 It renders the portlets from the `plone.footerportlets` portlet manager.
+
+## Boolean fields
+
+Since `zope.schema==6.1.0` all `zope.schema.Bool` fields need to have a `required=False` attribute,
+otherwise, you will not be able to save the form without marking the checkbox, which effectively turns that field to be always `True`.
```

Comment:
I think there is a use case to set `required=True` or not add the attribute `required=False`. For example:

- [ ] I agree to the Terms and Conditions of Service-o-rama.

The user is required to check the box to use Service-o-rama, else the form data would not be saved.

Reply:
At the schema level, `required=True` for a boolean field means that the field must have a value that is boolean (True or False) rather than missing (None).

If some widget library is treating it as meaning that a checkbox based on the field needs to be checked, that's a bug in that library, in my opinion.

For a checkbox that must have its value set to True, the way to do that at the schema level would be with a constraint (`constraint=lambda value: value is True`)

Reply:
OK, thanks for the explanation. I couldn't comprehend that issue's long discussion.